packages/webapp/src/hooks/useSyncs.tsx

```diff
@@ -12,3 +12,26 @@ export function useSyncs(queries: { env: string; connection_id: string; provider
 
     return { loading, error: error?.json, data: data, mutate };
 }
+
+export async function apiRunSyncCommand(
+    env: string,
+    body: {
+        command: RunSyncCommand;
+        schedule_id: string;
+        nango_connection_id: number;
+        sync_id: string;
+        sync_name: string;
+        provider: string;
+        empty_cache?: boolean;
+    }
+) {
+    const res = await apiFetch(`/api/v1/sync/command?env=${env}`, {
+        method: 'POST',
+        body: JSON.stringify(body)
+    });
+
+    return {
+        res,
+        json: await res.json()
```

Comment:
that's a problem for a future us :D 
no, but we don't handle this anywhere else, we probably need to test how it behaves and have a global fix. I guess rn it will throw and display the ErrorFallback component in the UI

Reply:
Yeah, was just curious. There's also probably an issue here of it returning a valid json body with a non-200 response code that could also be a challenge.